Sources/PubNub/Helpers/Streams/MultipartInputStream.swift

```diff
@@ -28,16 +28,18 @@
 import Foundation
 
 /// An `InputStream` that can combine multiple streams into a single stream
-class MultipartInputStream: InputStream {
+public class MultipartInputStream: InputStream {
```

Comment:
Do we need to make it public? Will be there any benefits for users to using it as `MultipartInputStream`? 

Earlier, this class was created to provide support of multipart-form body, which may consist of sections represented by separate streams with data.

Reply:
I thought it was your idea during the last review 🙂 The whole issue is about setting `Content-Length` when uploading a File, so `CryptorModule` method responsible for stream encryption has to return something more than `InputStream` only. Where do you prefer having encrypted stream length stored?